crates/wysiwyg/src/tests/test_formatting.rs

```diff
@@ -115,3 +115,14 @@ fn formatting_twice_adds_no_formatting() {
     }
     assert_eq!(tx(&model), input);
 }
+
+#[test]
+fn formatting_nested_format_nodes_and_line_breaks() {
```

Comment:
A few more tests would make me feel better - how about spanning multiple nodes, some of which contain BRs, and some of those BRs are not just at the top level?